liquibase-standard/src/main/java/liquibase/integration/spring/MultiTenantSpringLiquibase.java

```diff
@@ -76,252 +79,253 @@ public class MultiTenantSpringLiquibase implements InitializingBean, ResourceLoa
-		this.labelFilter = labelFilter;
-	}
-
-	public Map<String, String> getParameters() {
-		return parameters;
-	}
-
-	public void setParameters(Map<String, String> parameters) {
-		this.parameters = parameters;
-	}
-
-	public String getDefaultSchema() {
-		return defaultSchema;
-	}
-
-	public void setDefaultSchema(String defaultSchema) {
-		this.defaultSchema = defaultSchema;
-	}
-
-	public String getLiquibaseSchema() {
-		return liquibaseSchema;
-	}
-
-	public void setLiquibaseSchema(String liquibaseSchema) {
-		this.liquibaseSchema = liquibaseSchema;
-	}
-
-	public String getLiquibaseTablespace() {
-		return liquibaseTablespace;
-	}
-
-	public void setLiquibaseTablespace(String liquibaseTablespace) {
-		this.liquibaseTablespace = liquibaseTablespace;
-	}
-
-	public String getDatabaseChangeLogTable() {
-		return databaseChangeLogTable;
-	}
-
-	public void setDatabaseChangeLogTable(String databaseChangeLogTable) {
-		this.databaseChangeLogTable = databaseChangeLogTable;
-	}
-
-	public String getDatabaseChangeLogLockTable() {
-		return databaseChangeLogLockTable;
-	}
-
-	public void setDatabaseChangeLogLockTable(String databaseChangeLogLockTable) {
-		this.databaseChangeLogLockTable = databaseChangeLogLockTable;
-	}
-
-	public boolean isDropFirst() {
-		return dropFirst;
-	}
-
-	public void setDropFirst(boolean dropFirst) {
-		this.dropFirst = dropFirst;
-	}
-
-	public boolean isClearCheckSums() {
-		return clearCheckSums;
-	}
-
-	public void setClearCheckSums(boolean clearCheckSums) {
-		this.clearCheckSums = clearCheckSums;
-	}
-
-	public boolean isShouldRun() {
-		return shouldRun;
-	}
-
-	public void setShouldRun(boolean shouldRun) {
-		this.shouldRun = shouldRun;
-	}
-
-	public File getRollbackFile() {
-		return rollbackFile;
-	}
-
-	public void setRollbackFile(File rollbackFile) {
-		this.rollbackFile = rollbackFile;
-	}
-
-	@Override
-	public void setResourceLoader(ResourceLoader resourceLoader) {
-		this.resourceLoader = resourceLoader;
-	}
-
-	public List<String> getSchemas() {
-		return schemas;
-	}
-
-	public void setSchemas(List<String> schemas) {
-		this.schemas = schemas;
-	}
-
-	public DataSource getDataSource() {
-		return dataSource;
-	}
-	public void setDataSource(DataSource dataSource) {
-		this.dataSource = dataSource;
-	}
+        liquibase.setDropFirst(dropFirst);
+        liquibase.setClearCheckSums(clearCheckSums);
+        liquibase.setShouldRun(shouldRun);
+        liquibase.setRollbackFile(rollbackFile);
+        liquibase.setResourceLoader(resourceLoader);
+        liquibase.setDataSource(dataSource);
+        liquibase.setDefaultSchema(defaultSchema);
+        liquibase.setLiquibaseSchema(liquibaseSchema);
+        liquibase.setLiquibaseTablespace(liquibaseTablespace);
+        liquibase.setDatabaseChangeLogTable(databaseChangeLogTable);
+        liquibase.setDatabaseChangeLogLockTable(databaseChangeLogLockTable);
+        return liquibase;
+    }
+
+
+    public String getJndiBase() {
```

Comment:
As we are reviewing this class those methods could be replaced by lombok's @Setter and @Getter .

Reply:
Done, the setters and getters which have JavaDoc are preserved.

Reply:
Thanks @asashour !